prisma/seed.ts

```diff
@@ -16,11 +16,19 @@ async function seed() {
   temporaryExpDate.setDate(temporaryExpDate.getDate() + 7); // expiration for notifications
 
   // cleanup the existing database; no worries if it doesn't exist yet
+  await prisma.systemState.deleteMany().catch(() => {});
   await prisma.dnsRecord.deleteMany().catch(() => {});
   await prisma.challenge.deleteMany().catch(() => {});
   await prisma.certificate.deleteMany().catch(() => {});
   await prisma.user.deleteMany().catch(() => {});
 
+  await prisma.systemState.create({
+    data: {
+      unique: 'unique',
+      reconciliationNeeded: false,
```

Comment:
We could actually seed some DNS Records for users now, since they will get reflected in Route53 if we set this to true. That can happen in a follow up vs. here, just thinking out loud.

Reply:
Correct, reconciler will automatically push the (seed) data to route53